src/core/lib/security/context/security_context.h

```diff
@@ -178,4 +187,20 @@ grpc_auth_context* grpc_auth_context_from_arg(const grpc_arg* arg);
 grpc_auth_context* grpc_find_auth_context_in_args(
     const grpc_channel_args* args);
 
+namespace grpc_core {
+template <>
+struct ArenaContextType<SecurityContext> {
+  static void Destroy(SecurityContext* p) { p->~SecurityContext(); }
```

Comment:
Just checking: I assume that calling the base class's dtor directly will also call the subclass dtors, since the dtor is virtual?

Reply:
Yep!